packages/semantic-model/src/api-json.d.ts

```diff
@@ -260,6 +260,12 @@ export interface Ui5Property {
   optional?: boolean;
   static?: boolean;
   allowedFor?: string[];
+  [ui5 - metadata]?: {
```

Comment:
The UI5 team is also working on a semantic model for the `api.json` which could be reused in future...

Reply:
I guess this is just for information and nothing needs to be fixed for now